lib/active_interactor/organizer/callbacks.rb

```diff
@@ -141,11 +141,127 @@ def around_each_perform(*filters, &block)
         def before_each_perform(*filters, &block)
           set_callback(:each_perform, :before, *filters, &block)
         end
+
+        # Define a callback to call after all {Organizer::Organize::ClassMethods#organized organized}
+        # {ActiveInteractor::Base interactors'} {Interactor::Perform#perform #perform} methods have been called.
+        #
+        # @example
+        #  class MyInteractor1 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor1'
+        #    end
+        #  end
+        #
+        #  class MyInteractor2 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor2'
+        #    end
+        #  end
+        #
+        #  class MyOrganizer < ActiveInteractor::Organizer
+        #    after_all_perform :print_done
+        #
+        #    organized MyInteractor1, MyInteractor2
+        #
+        #    private
+        #
+        #    def print_done
+        #      puts 'Done'
+        #    end
+        #  end
+        #
+        #  MyOrganizer.perform
+        #  "MyInteractor1"
+        #  "MyInteractor2"
+        #  "Done"
+        #  #=> <MyOrganizer::Context>
+        def after_all_perform(*filters, &block)
+          set_callback(:all_perform, :after, *filters, &block)
+        end
+
+        # Define a callback to call around all {Organizer::Organize::ClassMethods#organized organized}
+        # {ActiveInteractor::Base interactors'} {Interactor::Perform#perform #perform} method calls.
+        #
+        # @example
+        #  class MyInteractor1 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor1'
+        #      sleep(1)
+        #    end
+        #  end
+        #
+        #  class MyInteractor2 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor2'
+        #      sleep(1)
+        #    end
+        #  end
+        #
+        #  class MyOrganizer < ActiveInteractor::Organizer
+        #    around_all_perform :print_time
+        #
+        #    organized MyInteractor1, MyInteractor2
+        #
+        #    private
+        #
+        #    def print_time
+        #      puts Time.now.utc
+        #      yield
```

Comment:
This seems logical for sure, and it's how RSpec does it with their around hooks. 

--

If someone forgot to call `yield` would it not run the organizer at all?

Reply:
Yep! But that could happen if you use any of the existing `around_*` callbacks with `yield`ing too